functions/test/wort/core_test.clj

```diff
@@ -4,4 +4,13 @@
 
 (deftest a-test
   (testing "FIXME, I fail."
-    (is (= 0 1))))
+    (is (= 1 1))))
+
+(deftest _get_words_from_key
```

Comment:
It's how you're importing. Two options:
* name your tests for what they test and split it up with fewer asserts (I like to only have one assert usually, but multiple is okay as long as it doesn't get out of hand)
* try replacing  the import on 3 with "wort.core" and then you can reference the function with wort.core/get_words_from_key (I think)

Up to you. Not really a blocker, just an annoyance.